docs/reference/commandline/ps.md

```diff
@@ -27,6 +27,7 @@ Options:
                         - since=(<container-name>|<container-id>)
                         - ancestor=(<image-name>[:tag]|<image-id>|<image@digest>)
                           containers created from an image or a descendant.
+                        - is-task=(true|false|any)
```

Comment:
Could it just be `--filter task`? We already have `ancestor` and not `is-ancestor`.